doc/release-notes.md

```diff
@@ -92,7 +92,9 @@ Deprecated or removed RPCs
 --------------------------
 
 - The `totalFee` option of the `bumpfee` RPC has been deprecated and will be
-  removed in 0.20.  To continue using this option start with
+  removed in 0.20. The `feeRate` option is now the preferred way to specify
+  the fee for the bump transaction.
```

Comment:
Nit: bumped

Reply:
I initially wrote it as 'bumped' but went with 'bump'. My thought was that the bumped transaction is the one we are replacing, and the bump transaction is the new one?

Reply:
`the fee for the new transaction.`

Reply:
thats better